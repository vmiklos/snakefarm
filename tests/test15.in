debug 1
out 1
time 100
load test15.map
addplayer 1
addsnake 1 1 0 1 1 1
18 0
17 0
endsnake
addplayer 2
addsnake 2 2 2 1 1 1
19 0
20 0
endsnake
comment # az elso lep eloszor, megeszi a masodik fejet, a masodik meghal, igy lepni sem fog
step
exit