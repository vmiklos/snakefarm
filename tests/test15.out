Event MapLoad test15.map
# az elso lep eloszor, megeszi a masodik fejet, a masodik meghal, igy lepni sem fog
Event Step 1
StepEvent Snake_Snake 2
StepEvent SnakeDie 2
End Step
Event Exit